### DISCUSSION TITLE
Starship prompt is not displaying full path despite setting truncation_length to 0

### DISCUSSION THREAD

**Discussion by a1b3rt** [starship/starship#5069] [Q&A]
Currently, my Starship prompt is only displaying the current directory. I would like it to display the full path of the current directory. According to the documentation, I should be able to achieve this by setting the truncation_length property of the directory module to 0. However, I have tried this and it did not have the desired effect. Can you help me understand why the truncation_length option is not working and how I can display the full path in my Starship prompt?

```
[directory]
truncation_length = 0
truncate_to_repo = true
fish_style_pwd_dir_length = 0
use_logical_path = true
format = "[$path]($style)[$read_only]($read_only_style) "
repo_root_format = "[$before_root_path]($before_repo_root_style)[$repo_root]($repo_root_style)[$path]($style)[$read_only]($read_only_style) "
style = "cyan bold"
disabled = false
read_only = " "
read_only_style = "red"
truncation_symbol = ""
home_symbol = "~"
use_os_path_sep = true
```

**Comment by mickimnet** [comment]
What happens if you set `truncate_to_repo = false`?
